From d7081520dde6814ecbc967ec9e15c39dc6d60692 Mon Sep 17 00:00:00 2001
From: Chris <meischris39@hotmail.com>
Date: Sun, 4 Oct 2026 08:42:33 +0100
Subject: [PATCH] [RetroPlayer] Remove leftover regex includes from the shader
 presets

They were only needed by the per-backend copies of GetShaderParameters().
---
 xbmc/cores/RetroPlayer/shaders/gl/ShaderPresetGL.cpp      | 2 --
 xbmc/cores/RetroPlayer/shaders/gles/ShaderPresetGLES.cpp  | 2 --
 xbmc/cores/RetroPlayer/shaders/windows/ShaderPresetDX.cpp | 1 -
 3 files changed, 5 deletions(-)

diff --git a/xbmc/cores/RetroPlayer/shaders/gl/ShaderPresetGL.cpp b/xbmc/cores/RetroPlayer/shaders/gl/ShaderPresetGL.cpp
index 9cf412a..cb37bbc 100644
--- a/xbmc/cores/RetroPlayer/shaders/gl/ShaderPresetGL.cpp
+++ b/xbmc/cores/RetroPlayer/shaders/gl/ShaderPresetGL.cpp
@@ -16,8 +16,6 @@
 #include "rendering/gl/RenderSystemGL.h"
 #include "utils/log.h"
 
-#include <regex>
-
 using namespace KODI::SHADER;
 
 CShaderPresetGL::CShaderPresetGL(RETRO::CRenderContext& context,
diff --git a/xbmc/cores/RetroPlayer/shaders/gles/ShaderPresetGLES.cpp b/xbmc/cores/RetroPlayer/shaders/gles/ShaderPresetGLES.cpp
index d29260e..6afeef1 100644
--- a/xbmc/cores/RetroPlayer/shaders/gles/ShaderPresetGLES.cpp
+++ b/xbmc/cores/RetroPlayer/shaders/gles/ShaderPresetGLES.cpp
@@ -16,8 +16,6 @@
 #include "rendering/gl/RenderSystemGL.h"
 #include "utils/log.h"
 
-#include <regex>
-
 using namespace KODI::SHADER;
 
 CShaderPresetGLES::CShaderPresetGLES(RETRO::CRenderContext& context,
diff --git a/xbmc/cores/RetroPlayer/shaders/windows/ShaderPresetDX.cpp b/xbmc/cores/RetroPlayer/shaders/windows/ShaderPresetDX.cpp
index 0b60653..33f8f11 100644
--- a/xbmc/cores/RetroPlayer/shaders/windows/ShaderPresetDX.cpp
+++ b/xbmc/cores/RetroPlayer/shaders/windows/ShaderPresetDX.cpp
@@ -17,7 +17,6 @@
 #include "utils/log.h"
 
 #include <cstddef>
-#include <regex>
 
 using namespace KODI::SHADER;
 
-- 
2.53.0

